task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -34,6 +34,13 @@ spec:
         #!/usr/bin/env bash
         source /utils.sh
         ### Try to extract binaries with configs > check binaries functionality > check opm validate ###
+        if [ ! -f ../sanity-inspect-image/image_inspect.json ] || [ -z $(cat ../sanity-inspect-image/image_inspect.json) ]; then
+          echo "File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image"
+          HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -f 1 -t 'File ../sanity-inspect-image/image_inspect.json is not generated correctly, please check HACBS_TEST_OUTPUT of task sanity-inspect-image!')"
```

Comment:
> `-f 1`
I'm not sure, if we should count error as failure, because this is error of the script itself it's not a failed test case

Reply:
Fixed. Thanks!